packages/core/src/services/agent.service.ts

```
@@ -30,15 +30,31 @@ export class AgentService {
     for (const workflow of agent.workflows) {
       try {
         await this.agentWorkflowsDatabaseService.create({
           agent_id: updatedAgent.id,
           workflow_id: workflow.id,
           status: Status.Active,
         });
-        visited[workflow.id] = true;
+        visitedWorkflows[workflow.id] = true;
+      } catch (error) {
+        console.log(error);
+        continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)